javascript/javascript3/week1/weather_api/weather.js

```diff
@@ -0,0 +1,60 @@
+const body = document.getElementById("div1");
```

Comment:
it is ok to name div1 but consider to make a better name to describe your tag.

Reply:
sure thanks